metis/lib/client/jsx/api/files_api.jsx

```diff
@@ -1,16 +1,19 @@
 import { json_get, json_delete, json_post } from '../utils/fetch';
 
+// TODO: These really ought to be refactored into a service class that knows about metis' domain
+// in various environments, rather than assuming the location of the currently viewed page contains
+// these endpoints.
 export const postRetrieveFiles = (project_name, bucket_name, folder_name) =>
-  json_get(`/${project_name}/list/${bucket_name}/${folder_name}`);
+  json_get(`${window.location.origin}/${project_name}/list/${bucket_name}/${folder_name}`);
```

Comment:
Required as our node-fetch stub in jest requires all fetch calls to include protocol and server (origin)